src/modm/platform/core/cortex/startup.c.in

```diff
@@ -190,5 +190,5 @@ Reset_Handler(void)
 	main();
 
 	modm_assert_debug(0, "core", "main", "exit");
-	while(1) ;
+	while (true) { continue; };
```

Comment:
```suggestion
	for(;;){}
```
What does MISRA say for this? 😏

Reply:
Checked with `cppcheck` and the `misra.py` addon. It would be fine.